src/Command/Initializer/Profiles/blank/Init.php

```diff
@@ -0,0 +1,542 @@
+                $templates['templates']['index']['replaces']['{imports}'] = "use Pimple\Container;\n" .
+                    "use Slim\Factory\AppFactory;";
+                $templates['templates']['index']['replaces']['{defineContainer}'] = <<<'BODY'
+// Instantiate Pimple Container
+$container = new Container();
+BODY;
+                $templates['templates']['index']['replaces']['{setContainer}'] = <<<'BODY'
+// Instantiate the app
+AppFactory::setContainer(new \Pimple\Psr11\Container($container));
+BODY;
+                break;
+            case 'other':
+                $templates['templates']['settings']['replaces']['{imports}'] = null;
+                $templates['templates']['settings']['replaces']['{argument}'] = '$container';
+                $templates['templates']['settings']['replaces']['{body}'] = "\n";
+
+                $templates['templates']['dependencies']['replaces']['{imports}'] = null;
+                $templates['templates']['dependencies']['replaces']['{argument}'] = '$container';
+                $templates['templates']['dependencies']['replaces']['{body}'] = "\n";
+
+                $templates['templates']['index']['replaces']['{containerVariable}'] = '$container';
+                $templates['templates']['index']['replaces']['{imports}'] = "use Slim\Factory\AppFactory;";
+                $templates['templates']['index']['replaces']['{defineContainer}'] = <<<'BODY'
+// TODO: Instantiate you'r Dependency Container
+$container = null;
+BODY;
+                $templates['templates']['index']['replaces']['{setContainer}'] = <<<'BODY'
+// Instantiate the app
+// TODO: Uncomment the line below if you created an instance of Dependency Container
+//AppFactory::setContainer($container);
+BODY;
+                break;
+        }
+
+        $templates['templates']['settings']['replaces']['{appName}'] = $projectDirectory;
+
+        $templates['templates']['middleware']['replaces']['{imports}'] = "\nuse Slim\App;\n";
+        $templates['templates']['middleware']['replaces']['{argument}'] = 'App $app';
+        $templates['templates']['middleware']['replaces']['{body}'] = "\n";
+
+        // Replace tokens in templates and write to destination files.
+        foreach ($templates['templates'] as $k => $template) {
+            foreach ($template['replaces'] as $token => $replace) {
+                $template['template'] = str_replace($token, $replace, (string)$template['template']);
+            }
+
+            file_put_contents(
+                $templates['destinationPrefix'] . $template['destinationFile'],
+                $template['template']
+            );
+        }
+
+        return $this->writeToComposerJson($directoryFullPath, $composerJsonContent);
+    }
+
+    /**
+     * Collect dependencies from user input.
+     *
+     * @return array<array>
+     */
+    protected function askDependencies(): array
+    {
+        $requestResponse = null;
```

Comment:
As mentioned in comment above, let's change all instances of `requestResponse` to `psr7`